pkg/controller/iam/partialpolicy/iampartialpolicy_controller.go

```diff
@@ -186,8 +191,10 @@ func (r *ReconcileIAMPartialPolicy) Reconcile(ctx context.Context, request recon
 	if err != nil {
 		return reconcile.Result{}, err
 	}
-	logger.Info("successfully finished reconcile", "resource", request.NamespacedName, "time to next reconciliation", jitteredPeriod)
-	return reconcile.Result{RequeueAfter: jitteredPeriod}, nil
+	requeueDelay := r.requeueRateLimiter.When(request)
```

Comment:
qq: is this the rate limiter with 5 qps limit that computes the delay so that our requeue will max out at 5qps?